py3status/modules/xrandr_rotate.py

```diff
@@ -89,7 +77,7 @@ def _apply(self):
         outputs = [self.screen] if self.screen else self._get_all_outputs()
         for output in outputs:
             cmd = 'exec xrandr --output ' + output + ' --rotate ' + rotation
-            Popen(['i3-msg', cmd], stdout=PIPE)
+            self.py3.command_run(['i3-msg', cmd])
```

Comment:
If this is a necessary, we could try `i3-msg exec` for other `xrandr` since it's just calling the command directly.... I think this works okay on my old laptop, but not my `DP-2+DP-3 monitors` computer. Also, if you add socket in 3.6, then we might want the new change anyway. Let me know.